src/main/java/me/crafter/mc/lockettepro/LocketteProAPI.java

```diff
@@ -462,7 +464,7 @@ public static void toggleDoor(Block block, boolean open) {
         org.bukkit.block.data.Openable openablestate = (org.bukkit.block.data.Openable) block.getBlockData();
         openablestate.setOpen(open);
         block.setBlockData(openablestate);
-        block.getWorld().playSound(block.getLocation(), open ? Sound.BLOCK_WOODEN_DOOR_OPEN : Sound.BLOCK_WOODEN_DOOR_CLOSE, 100,1);
+        block.getWorld().playSound(block.getLocation(), open ? Sound.BLOCK_WOODEN_DOOR_OPEN : Sound.BLOCK_WOODEN_DOOR_CLOSE, 1, 1);
```

Comment:
😱

Reply:
\*[flashy-thing](http://2.bp.blogspot.com/-pWmrlshMj9M/UBlIQRZYlkI/AAAAAAAAAI8/8Qm9abuzmfw/s1600/will-smith-men-in-black-3-flash.jpg)s you\*
You saw nothing. You will live a happy life and fix the Minigames-Plugin.